docs/modules/ROOT/pages/index.adoc

```diff
@@ -5,7 +5,7 @@ The intention of this repository is to provide a simple, out-of-the-box solution
 * An additional requirement is to ensure we can integrate into existing observability pipelines people may already have as easily as possible.
 * This must all support being deployed on-premise and on cloud platforms with minimal change.
 * Any bespoke software must be minimal and ideally just restricted to configuration of generic tools.
-* We must support customer configuration of what is "important" to monitor in their clusters although with best practice defaults provided.
+* We support customer configuration of what is "important" to monitor in their clusters by providing best-practice defaults.
```

Comment:
I'm not sure that's what was originally meant by that sentence - the way I read the original is, more or less, "we should have defaults, but should also support overriding them", while the new version is closer to "we have defaults"

Reply:
@markspolakovs yeah you're right let me fix it. Thanks!